### PR TITLE
ticket-9-delete-comments

### PR DESCRIPTION
**Pull request by natha7**

### CODE REVIEW

File: src/components/CommentCard.jsx [hunk 1]
```diff
@@ -1,14 +1,50 @@
...
+      return true;
+    });
+    deleteCommentByCommentId(comment.comment_id).then(() => {
+      setComments((currentComments) => {
+        const indexToSplice = currentComments.findIndex((currComment) => {
+          return currComment.comment_id === comment.comment_id;
+        });
+        const splicedComments = currentComments.toSpliced(indexToSplice, 1);
+
+        return [...splicedComments];
```
Comment (nc-mark-t) [inline src/components/CommentCard.jsx:23]: You could just filter and you wouldn't need to iterate through the array twice or in this case 3 times

File: src/components/CommentCard.jsx [hunk 2]
```diff
@@ -1,14 +1,50 @@
...
+      });
+    });
+  }
 
   return (
     <article className="comment-card">
       <p className="comment-author">{comment.author}</p>
-      <p className="comment-body">{comment.body}</p>
+      <section className="comment-body">
+        {comment.body.split("\n").map((commentSegment, index) => {
```
Comment (nc-mark-t) [inline src/components/CommentCard.jsx:35]: Why is this necessary?

File: src/components/CommentCard.jsx [hunk 3]
```diff
@@ -1,14 +1,50 @@
...
+        return false;
+      });
+    });
+  }
 
   return (
     <article className="comment-card">
       <p className="comment-author">{comment.author}</p>
-      <p className="comment-body">{comment.body}</p>
+      <section className="comment-body">
```
Comment (nc-mark-t) [inline src/components/CommentCard.jsx:34]: Is the section just for the CSS? You can use a div if just for styling

**Review by nc-mark-t — COMMENTED** [review]
The functionality works well, see my comments for potential improvements

File: src/components/CommentCard.jsx (hunk 2 above)
Comment (natha7) [reply src/components/CommentCard.jsx:35]: Hi Mark, any enters in a posted comments text area were being joint together so this was to allow the comments to be formatted in paragraph styles
